ambari-agent/pom.xml

```diff
@@ -53,6 +53,7 @@
     <resourcesFolder>${ambari.server.module}/src/main/resources</resourcesFolder>
     <customActionsLocation>${target.cache.dir}/custom_actions</customActionsLocation>
     <empty.dir>src/packages/tarball</empty.dir> <!-- any directory in project with not very big amount of files (not to waste-load them) -->
+    <hadoop.version>3.3.4</hadoop.version>
```

Comment:
the hadoop version bump is fixed by [AMBARI-26323](https://github.com/apache/ambari/pull/3955/files#top)